README.md

```diff
@@ -1,51 +1,94 @@
-You will also see the example output files which are generally TSV or PNG files.
-All the data used in these examples are deposited into the `data/` 
-directory and are referenced as such in the R Notebooks.
+#### gct format conversion
+Convert a gene expression tab separated values (tsv) file provided
+into a gene cluster text (gct) file for use in GenePattern notebooks.
+In order to create a gct formatted file from a tsv refine.bio data file,
+reference the
+[`create_gct_file.R` script](https://github.com/AlexsLemonade/refinebio-examples/blob/master/scripts/create_gct_file.R), followed by `-f` argument with the name
+of the file in your current directory that you would like to convert.
+Note: This script requires optparse library. If optparse library is not
+installed, this script will install it for you.
+
+##### Example of usage in command line:  
+ ```bash
+ $ Rscript create_gct_file.R -f GSE111111.tsv -o outputfilename -r
+ ```
+ Options:   
+`-f` :name of the file in your current directory that you would like to convert.    
+`-r` :file of the same name as the output will be rewritten (*optional*)       
+`-o` :name for the output file (*optional*)       
+
+Optionally you can designate the name of the output file by adding an `-o`
+argument.
+The "gct" suffix will be added if you do not add it yourself.
+Be sure to either have the script and input file in your current working
+directory, or put type out the full directory path for the script and/or input
+file. eg `/users/Bob/Desktop/create_gct_file.R`
+
+#### CLS format file creation
+CLS formatted files tell information regarding the groups or phenotype of the
+different samples and are necessary for doing gene expression differential
+analysis using GenePattern's notebooks. If you've already created a gct format
+file from your data, you can create a a CLS format using GenePattern's online
+[CLSFileCreator](http://software.broadinstitute.org/cancer/software/genepattern/modules/docs/ClsFileCreator/4)
 
-# Table of Contents
+# Table of Contents  
+
+1. [Clustering data](https://github.com/AlexsLemonade/refinebio-examples/blob/master/clustering)  
 
-1. [Clustering data](https://github.com/AlexsLemonade/refinebio-examples/blob/master/clustering) 
-  
 2. [Annotation using Ensembl IDs](https://github.com/AlexsLemonade/refinebio-examples/blob/master/ensembl-id-convert)  
-  
-3. [Differential Expression Analyses](https://github.com/AlexsLemonade/refinebio-examples/blob/master/differential-expression) 
-  
+
+3. [Differential Expression Analyses](https://github.com/AlexsLemonade/refinebio-examples/blob/master/differential-expression)  
+
 4. [Ortholog Mapping](https://github.com/AlexsLemonade/refinebio-examples/blob/master/ortholog-mapping)  
-  
-5. [Other scripts](https://github.com/AlexsLemonade/refinebio-examples/blob/master/scripts) 
 
+5. [Batch Correction with ComBat](https://github.com/AlexsLemonade/refinebio-examples/blob/master/batch_correction)
```

Comment:
This should get added in the same PR where the ComBat example gets added not before because it would introduce a broken link.